101-data-share-share/azuredeploy.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "account_name": {
```

Comment:
you should be able to remove all the values from here once you have defaultValues in the template

Reply:
> you should be able to remove all the values from here once you have defaultValues in the template

for the deployed locations we only support part of the regions of public and gov clouds.  Is there a way to keep them separate without having a parameter file?

Reply:
@bmoore-msft

Reply:
Westus is not one of our support regions as of now, so I've added a 'allowedValues' for location with all supporting regions from public and usgov values. Please have a review for the new iteration. Thanks~